client/console/src/main/resources/org/apache/syncope/client/console/panels/GroupDirectoryPanel_fr_CA.properties

```diff
@@ -16,3 +16,4 @@
 # under the License.
 any.edit=Modifier ${anyTO.type} ${anyTO.name}
 group.members=${right} membres de ${left.name}
+auditHistory.title=${anyTO.type} ${anyTO.fullGroupName != "-" ? anyTO.fullGroupName : anyTO.name} histoire
```

Comment:
Please do a straight commit for this fix, both for `2_1_X` and `master`, no need to include it in this PR.